src/abstractdataframe/abstractdataframe.jl

```diff
@@ -106,18 +106,18 @@ Set column names
 names!(df::AbstractDataFrame, vals)
 ```
 
-### Arguments
+**Arguments**
```

Comment:
I prefer headings, too, but they don't work well with MkDocs. The headings end up in the contents bar on the left, and it clutters it up in a confusing way.